queues.md

```diff
@@ -816,6 +816,42 @@ Likewise, if the `after_commit` configuration option is set to `true`, you may i
 
     ProcessPodcast::dispatch($podcast)->beforeCommit();
 
+<a name="specifying-commit-dispatch-behavior-on-the-job-class"></a>
+#### Specifying Commit Dispatch Behavior On The Job Class
+
+Alternatively, if you want to specify the commit dispatch behavior inside the job class, you can do so by implementing the `ShouldQueueAfterCommit` interface instead of `ShouldQueue`:
+    <?php
```

Comment:
I think a newline is necessary here (haven't double-checked), at the very least for consistency
```suggestion
Alternatively, if you want to specify the commit dispatch behavior inside the job class, you can do so by implementing the `ShouldQueueAfterCommit` interface instead of `ShouldQueue`:

    <?php
```